plumbing/transport/ssh/common.go

```diff
@@ -11,7 +11,16 @@ import (
 )
 
 // DefaultClient is the default SSH client.
-var DefaultClient = common.NewClient(&runner{})
+var DefaultClient = NewClient()
+
+// NewClient creates a new SSH client with the given options.
+func NewClient(opts ...ClientOption) transport.Transport {
```

Comment:
That's right. The only difference is that global options could not override session values (e.g. adding a global override on every `HostKeyCallback`, such as an interceptor). Is that ok?